pkg/processor/runtime/rpc/abstract.go

```diff
@@ -210,20 +157,8 @@ func (r *AbstractRuntime) Restart() error {
 		return err
 	}
 
-	// Send error for current event (non-blocking)
-	select {
-	case r.resultChan <- &batchedResults{
-		results: []*result{{
-			StatusCode: http.StatusRequestTimeout,
-			err:        errors.New("Runtime restarted"),
-		}},
-	}:
-
-	default:
-		r.Logger.Warn("Nothing waiting on result channel during restart. Continuing")
-	}
+	_ = r.connectionManager.Stop()
```

Comment:
Why ignore the error?
I saw that currently the socketAllocator always returns `nil`, but this function is unaware of it, as it calls the interface.